Cargo.toml

```diff
@@ -18,12 +19,12 @@ futures = "0.3.28"
 subxt = "0.32.1"
 tokio = { version = "1", features = ["full", "rt-multi-thread"] }
 
-parity-scale-codec = "3.6.8" # required by subxt
+parity-scale-codec = "3.6.5" # required by subxt
```

Comment:
`3.6.8` couldn't be resolved. 😧 Neither could `3.6.8-alpha.1`.

Reply:
It was yanked it seems: https://crates.io/crates/parity-scale-codec/versions.